app/Providers/AppServiceProvider.php

```diff
@@ -13,7 +13,14 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        //
+        $models = [
+            'User',
+        ];
+        foreach ($models as $model) {
+            $contracts = 'App\Repositories\Contracts\\' . $model . 'RepositoryInterface';
+            $eloquents = 'App\Repositories\Eloquents\\' . $model . 'Repository';
+            $this->app->bind($contracts, $eloquents);
+        }
```

Comment:
tại sao phải bind ở đây?

Reply:
- khi chạy ứng dụng sẽ load các provider trong đó có AppServiceProvider kế thừa từ Service Provider. viết đè hàm register để có thể sử dụng trong toàn project ạ